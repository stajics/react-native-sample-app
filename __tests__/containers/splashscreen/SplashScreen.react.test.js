import 'react-native';
import React from 'react';
import { SplashScreen } from '../../../app/containers/splashscreen/SplashScreen.react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'; // eslint-disable-line

it('renders correctly', () => {
  const splashscreen = renderer.create(
    <SplashScreen
      appTitle={'TEST APP TITLE'}
    />
  ).toJSON();
  expect(splashscreen).toMatchSnapshot();
});
