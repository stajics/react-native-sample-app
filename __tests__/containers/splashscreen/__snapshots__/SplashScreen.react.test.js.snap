exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }>
    TEST APP TITLE
  </Text>
  <Image
    source={
      Object {
        "uri": "http://image.flaticon.com/teams/new/1-freepik.jpg",
      }
    }
    style={
      Object {
        "height": 200,
        "width": 200,
      }
    } />
</View>
`;
